### ISSUE TITLE
`value` is ignored

### ISSUE THREAD

**Issue by blythed** [lf1-io/padl#241]
Saving code:

```python
f = open('checkpoints/imagenet/index_to_name.json')
LABELS = value(json.load(f))
f.close()
```

Serialized code:

```python
f = open('checkpoints/imagenet/index_to_name.json')
LABELS = value(json.load(f))
```

This is incorrect, the `value` should mean that the `LABELS` variable is loaded from the saved directory.

**Comment by blythed** [comment]
Even smaller example:

```python
from padl import transform, value, save

x = [1, 2, 3]
LABELS = value(x)


@transform
def test(x):
    return LABELS[x]

save(test, 'test.padl', force_overwrite=True)
```

**Comment by wuhu** [comment]
@blythed 
I tried reproducing with the "smaller example", but it worked. Here's the `test.padl/transform.py`:

```
from padl import transform


LABELS = [1, 2, 3]


@transform
def test(x):
    return LABELS[x]


_pd_main = test
```

could you provide more info about the circumstances?

**Comment by blythed** [comment]
I tried with `IPython` and regular `python` on Mac OSX. Executed exactly this code in a small script. Anything else I can provide?

**Comment by wuhu** [comment]
I found that it didn't work for me when using regular python. Fixed it in #251. Ipython was working for me before.
@blythed could you please check if the PR fixes the issue for you?
